app/Models/Asset.php

```diff
@@ -15,4 +15,13 @@ public function investment(): BelongsTo
     {
         return $this->hasMany(Investment::class);
     }
+    /**
+     * Get the withdrawal associated with the Asset
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasOne
+     */
+    public function withdrawal(): HasOne
```

Comment:
Change this relationship to hasMany.

Many withdrawals can belong to an Asset. So its a one-to-many  relationship.